tcms/testruns/templates/testruns/get.html

```diff
@@ -256,7 +256,19 @@ <h5><span class="test-executions-count"></span> Results</h5>
                                                 <div>
                                                     <ul class="test-execution-hyperlinks"></ul>
                                                     <template id="link-entry">
-                                                        <li><span class="link-icon"></span><a class="link-url" href="${link.url}">${link.name || link.url}</a></li>
+                                                        <li>
+                                                            <span class="link-icon"></span>
+                                                            <a class="link-url" href="${link.url}">${link.name || link.url}</a>
```

Comment:
`${link.url}` not replaced when rendered. For a bug where I can fetch the details the URL is displayed fine but leads to `http://127.0.0.1:8000/runs/28/$%7Blink.url%7D`